expan/core/experiment.py

```diff
@@ -190,36 +190,51 @@ def filter(self, kpis, percentile=99.0, threshold_type='upper'):
 
         self.data = self.data[flags == False]
 
-    def sga(self, feature_name_to_bins):
+    def sga(self, feature_name_to_bins, time_interval=None):
```

Comment:
Shouldn't time binning be a special case of binning in general? If so, the information about it should be contained in `feature_name_to_bins`

Reply:
afair we discussed with the product the possibility to choose the time interval on which to run the experiment. So, I consider that as one more bin in the sga. Doing some random bins on data can be not very useful for the stakeholder, because the interval he is interested in may not be covered with the bins. Probably by day binning could be interesting, but that can be done just by grouping by day. I can discuss that with Adrien.

Reply:
One potential way to go would be to map the time column into an additional column with Unix time (https://en.wikipedia.org/wiki/Unix_time) and, Unix time being just an `int` of seconds, apply numerical binning on that column.

Reply:
sure, that will be binning by time since treatment, which can be added and is useful I think. I can add that. But I am not sure it is good to create additional column in data in Expan, but probably do that in Expan service.

Reply:
Another idea would be to create an additional column with date only (no hours:minutes:seconds) an group the data frame by that column...

Reply:
correct me if I am wrong, but date doesn't have timestamp currently, the format is 'YYYYDDMM' or 'YYYY-DD-MM', so, the categorical binning already can make bins on that column. But I can add unit tests for checking that.